FirebaseCore/Internal/Sources/HeartbeatLogging/HeartbeatLoggingTestUtils.swift

```diff
@@ -0,0 +1,140 @@
+// Copyright 2021 Google LLC
```

Comment:
```suggestion
// Copyright 2023 Google LLC
```

Reply:
Hmm, does a renamed file need an updated copyright year? @paulb777

Reply:
Nope, it can keep the 2021